cl/search/models.py

```diff
@@ -1923,6 +1995,27 @@ class OpinionCluster(models.Model):
         db_index=True,
         default=False,
     )
+    filepath_local = models.FileField(
+        help_text="Path to Harvard json local storage.",
+        max_length=1000,
+        blank=True,
+    )
+    sha1 = models.CharField(
+        help_text="unique ID for the document, as generated via SHA1 of the "
+        "binary file or text data",
+        max_length=40,
+        db_index=True,
+    )
+    page_count = models.IntegerField(
+        help_text="The number of pages in the opinion cluster.",
+        blank=True,
+        null=True,
+    )
+    image_missing = models.BooleanField(
```

Comment:
Are we using this or do we plan to? Bit of a curious one.

Reply:
Yes, we need it to know which cases to re-scrape at a later date.